app/com/gu/viewer/proxy/PreviewProxy.scala

```diff
@@ -96,6 +114,13 @@ class PreviewProxy @Inject() (proxyClient: Proxy) extends Loggable {
     }
   }
 
+  def proxyPost(request: PreviewProxyRequest): Future[Result] = ProxyResult.resultFrom {
+    request.session.sessionCookie -> request.session.authCookie match {
```

Comment:
Both of the outcomes (doPreviewProxyPost and doPreviewAuth) log, but agree makes sense to add in here